xml/art_sle_ha_pmremote.xml

```diff
@@ -464,21 +649,17 @@ ssh: connect to host &wsII; port 3121: Connection refused</screen>
        linkend="st.ha.pmremote.integrate.guestsintocluster.virsh-dump"/>:
      </para>
      <screen>&prompt.root;<command>crm</command> configure
-&prompt.crm.conf;<command>primitive</command> vm-&wsII; ocf:heartbeat:VirtualDomain \
+&prompt.crm.conf;<command>primitive</command> vm-&node4; ocf:heartbeat:VirtualDomain \
   params hypervisor="qemu:///system" \
-         config="/etc/pacemaker/&wsII;.xml" \
-  meta remote-node=&wsII;</screen>
+         config="/etc/pacemaker/&node4;.xml" \
+  meta remote-node=&node4; \
+  op monitor interval=10</screen>
```

Comment:
[NOTE]
======
Pacemaker will automatically monitor pacemaker_remote connections for failure,
so it is not necessary to create a recurring monitor on the VirtualDomain
resource.